action/protocol/staking/candidate_center.go

```diff
@@ -46,17 +38,23 @@ func (m CandidateCenter) ContainsName(name string) bool {
 }
 
 // ContainsOwner returns true if the map contains the candidate by owner
-func (m CandidateCenter) ContainsOwner(owner address.Address) bool {
+func (m CandidateCenter) ContainsOwner(owner string) bool {
```

Comment:
you don't need to change the apis, just change the internal map to use string and convert in the functions.

Reply:
yeah, fixed in latest push